client/src/pages/Courses/Courses.js

```diff
@@ -23,94 +23,103 @@ const Courses = () => {
-                                setSelectedCourse({});
-                                setIsAddingCourse(true);
-                            }}
-                        >
-                            <Text textAlign="center" color={c.white}>
-                                + Add Course
-                            </Text>
-                        </Button>
-                        <Button
-                            variantColor="blue"
-                            backgroundColor="#162971"
-                            onClick={() => {
-                                setPopulateFromUniAPI(true);
-                            }}
-                        >
-                            <Text textAlign="center" color="white">
-                                + Create From Uni API
-                            </Text>
-                        </Button>
-                        <PopulateAPIModal
-                            isOpen={populateFromUniAPI}
-                            onClose={() => setPopulateFromUniAPI(false)}
-                            confirm={createAllCoursesFromUniAPI}
-                        ></PopulateAPIModal>
-                    </Flex>
+                <Flex justify="flex-end" direction="row" p={2} mr="4%">
+                    <Button
+                        color={c.white}
+                        bg={c.midnightBlue}
+                        border="1px"
+                        borderColor={c.white}
+                        _hover={{bg: c.lightBlue, borderColor: c.lightBlue}}
+                        onClick={() => {
+                            setSelectedCourse({});
+                            setIsAddingCourse(true);
+                        }}
+                        mr="10px"
+                    >
+                        + Create
+                    </Button>
+                    
+                    <Button
+                        color={c.white}
+                        bg={c.midnightBlue}
+                        border="1px"
+                        borderColor={c.white}
+                        _hover={{bg: c.lightBlue, borderColor: c.lightBlue}}
+                        onClick={() => {
+                            setPopulateFromUniAPI(true);
+                        }}
+                    >
+                        + Create From Uni API
+                    </Button>
+
+                    <PopulateAPIModal
+                        isOpen={populateFromUniAPI}
+                        onClose={() => setPopulateFromUniAPI(false)}
+                        confirm={createAllCoursesFromUniAPI}
+                    />
                 </Flex>
 
+                <SearchBar searchCategory="Courses" value={searchTerm} onChange={(e) => setSearchTerm(e.target.value)} />
+
                 <Flex p={4}>
                     <Table
                         columns={columns}
-                        data={data}
+                        data={data.map(d => ({ ...d, semester: d.semester.join(", ") }))}
```

Comment:
u can leave the data untouched and change the semester accessor to accessor: d => d.semester.join(", "), in columns

Reply:
But the Students page uses the same table

Reply:
arent the defined columns different

Reply:
![image](https://user-images.githubusercontent.com/39323194/94354452-addf6700-00d7-11eb-8fc5-cc2b057f6893.png)
![image](https://user-images.githubusercontent.com/39323194/94354461-dbc4ab80-00d7-11eb-9571-be7ecda37cb6.png)
this seems to work

Reply:
Agree that we shouldn't be directly manipulating the data ^

Reply:
Ok i'll try it